src/components/Article-comments.jsx

```diff
@@ -0,0 +1,81 @@
+import { React, useEffect } from "react";
```

Comment:
Unnecessary 'React' in the import! (think this might be applicable in some other cases, too...